src/js/components/stock-movement-wizard/combined-shipments/SendMovementPage.jsx

```diff
@@ -789,6 +789,20 @@ class SendMovementPage extends Component {
                           if (document.hidden) {
                             return null;
                           }
+
+                          // Some documents have multiple download options so list them all out.
+                          if (document.downloadOptions) {
+                            return _.map(document.downloadOptions, (downloadOption) => (
+                              <DocumentButton
+                                link={downloadOption.uri}
+                                buttonTitle={downloadOption.name}
+                                {...document}
+                                key={idx}
```

Comment:
I'm not sure if we even need this line. It works fine with it, but I'm not certain what it's actually doing (I just copied it from the below button). Maybe someone with more frontend experience can answer

Reply:
https://legacy.reactjs.org/docs/lists-and-keys.html#keys